app/src/main/res/layout/activity_thermometer.xml

```diff
@@ -1,36 +1,137 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:theme="@style/AppTheme.AppBarOverlay">
+        android:layout_margin="@dimen/card_margin">
 
-    </android.support.design.widget.AppBarLayout>
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:baselineAligned="false">
+
+            <LinearLayout
+                android:layout_width="0dp"
+                android:layout_height="match_parent"
+                android:layout_weight="3"
+                android:gravity="center_horizontal|center_vertical"
+                android:orientation="vertical">
+
+                <TextView
+                    android:id="@+id/label_thermo_sensor"
+                    style="@style/lux_meter_stat_heading"
+                    android:paddingBottom="5dp"
+                    android:textSize="@dimen/lux_sensor_label_title_size" />
+
+                <TextView
+                    android:id="@+id/label_thermo_stat_max"
+                    style="@style/lux_meter_stat_heading"
+                    android:text="@string/max_thermo" />
+
+                <TextView
+                    android:id="@+id/thermo_max"
+                    style="@style/lux_meter_stat_display" />
+
+                <TextView
+                    android:id="@+id/label_thermo_stat_min"
+                    style="@style/lux_meter_stat_heading"
+                    android:text="@string/min_lx" />
+
+                <TextView
+                    android:id="@+id/thermo_min"
+                    style="@style/lux_meter_stat_display" />
+
+                <TextView
+                    android:id="@+id/label_thermo_stat_avg"
+                    style="@style/lux_meter_stat_heading"
+                    android:text="@string/avg_thermo" />
+
+                <TextView
+                    android:id="@+id/thermo_avg"
+                    style="@style/lux_meter_stat_display" />
+
+            </LinearLayout>
 
-    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        xmlns:tools="http://schemas.android.com/tools"
+            <LinearLayout
+                android:layout_width="@dimen/length_0dp"
+                android:layout_height="match_parent"
+                android:layout_weight="5"
+                android:orientation="vertical">
+
+                <com.github.anastr.speedviewlib.PointerSpeedometer
+                    android:id="@+id/thermo_meter"
+                    android:layout_width="match_parent"
+                    android:layout_height="@dimen/speedometer_height"
+                    android:padding="15dp"
+                    app:sv_speedTextColor="@color/black"
+                    app:sv_speedTextPosition="BOTTOM_CENTER"
+                    app:sv_speedTextSize="@dimen/lux_display_font_size"
```

Comment:
Can you make separate variables for thermometer replacing the `lux`s?

Reply:
No . I kept it the same knowigly because the dimen value are going to be same eventually in UI but I will change it if that is better

Reply:
Yes, all needs to be changed. The real reason why we are using resources instead of hard-coded values is that, it will be easier and practical to make modifications from one place. Putting luxmeter dimensions is in fact similar to hard-coding.